multiversx_sdk_cli/cli_wallet.py

```diff
@@ -28,41 +41,50 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
         "new",
         "Create a new wallet and print its mnemonic; optionally save as password-protected JSON (recommended) or PEM (not recommended)"
     )
-    sub.add_argument("--json",
-                     help="whether to create a json key file", action="store_true", default=False)
-    sub.add_argument("--pem",
-                     help="whether to create a pem key file", action="store_true", default=False)
-    sub.add_argument("--output-path",
-                     help="the output path and base file name for the generated wallet files (default: %(default)s)", type=str, default="./wallet")
-    sub.set_defaults(func=new_wallet)
+    sub.add_argument("--json", help="DEPRECATED, replaced by --format=keystore-mnemonic", action="store_true", default=False)
+    sub.add_argument("--pem", help="DEPRECATED, replaced by --format=pem", action="store_true", default=False)
+    sub.add_argument("--output-path", help="DEPRECATED, replaced by --outfile", type=str)
+    sub.add_argument("--format", choices=WALLET_FORMATS, help="the format of the generated wallet file (default: %(default)s)", default=None)
+    sub.add_argument("--outfile", help="the output path and base file name for the generated wallet files (default: %(default)s)", type=str)
+    sub.add_argument("--address-hrp", help=f"the human-readable part of the address, when format is {WALLET_FORMAT_KEYSTORE_SECRET_KEY} or {WALLET_FORMAT_PEM} (default: %(default)s)", type=str, default="erd")
+    sub.set_defaults(func=wallet_new)
 
     sub = cli_shared.add_command_subparser(
         subparsers,
         "wallet",
         "derive",
-        "Derive a PEM file from a mnemonic or generate a new PEM file (for tests only!)"
+        "DEPRECATED COMMAND, replaced by 'wallet convert'"
     )
-    sub.add_argument("pem",
-                     help="path of the output PEM file")
-    sub.add_argument("--mnemonic", action="store_true",
-                     help="whether to derive from an existing mnemonic")
-    sub.add_argument("--index",
-                     help="the account index", type=int, default=0)
-    sub.set_defaults(func=generate_pem)
+    sub.add_argument("pem", help="path of the output PEM file")
+    sub.add_argument("--mnemonic", action="store_true", help="whether to derive from an existing mnemonic")
+    sub.add_argument("--index", help="the address index", type=int, default=0)
+    sub.set_defaults(func=wallet_derive_deprecated)
+
+    sub = cli_shared.add_command_subparser(
+        subparsers,
+        "wallet",
+        "convert",
+        "Convert a wallet from one format to another"
+    )
+
+    sub.add_argument("--infile", help="path to the input file")
+    sub.add_argument("--outfile", help="path to the output file")
+    sub.add_argument("--in-format", required=True, choices=WALLET_FORMATS, help="the format of the input file")
```

Comment:
can all the possible options be listed in the help? are they by default?

Reply:
They are listed in the help in the following format:

https://github.com/multiversx/mx-sdk-py-cli/blob/wallet-07/CLI.md#walletconvert